pyproject.toml

```diff
@@ -1,8 +1,8 @@
-[tool.poetry]
+[project]
 name = "xortool"
-version = "1.0.2"
+version = "1.1.0"
 description = "A tool to analyze multi-byte xor cipher"
-authors = ["hellman"]
+authors = [{ name = "DawnMagnet", email = "axccjqh@qq.com" }]
```

Comment:
I think this is a bit too much, you can create list of contributors somewhere and add yourself (incl. repo's readme if you want).